Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -469,13 +480,24 @@ import Foundation
         components.path = "/graphql"
         return components.url
     }
+    
+    func paypalAPIURL(forEnvironment environment: String) -> URL? {
```

Comment:
```suggestion
    func payPalAPIURL(forEnvironment environment: String) -> URL? {
```

Reply:
Good catch, fixed in 335b594f5f61bfec610f8897c3fa96aa708c92f1